src/components/MerchantPayout/VoucherStatusModal/VoucherStatusModal.tsx

```diff
@@ -129,15 +145,17 @@ export const VoucherStatusModal: FunctionComponent<VoucherStatusModal> = ({
     card = (
       <InvalidCard
         title={title}
-        details="Please log an appeal request"
+        details={i18n.t("errorMessages.notEligibleScreen.logAppeal")}
```

Comment:
Eh are all these supposed to be in JJ's pr? I see about 42 instances of `errorMessages.XXX`